create.sh

```diff
@@ -49,10 +49,12 @@ name: openshift
 location: eastus
 properties:
   openShiftVersion: "$DEPLOY_VERSION"
+  FQDN: $RESOURCEGROUP.eastus.cloudapp.azure.com
```

Comment:
this is all backwards - FQDN is the r/o field by which the RP indicates the chosen name.

Reply:
so whats wrong here?  
FQDN: mjudeikis.eastus.cloudapp.azure.com corresponds to random RP generated name, which should point to `ip-apiserver`. To fake it, we use loadbalancer default DNS name for now.

Reply:
This will be auto-allocated by RP:
https://github.com/jim-minter/acs-engine/blob/master/pkg/api/osa/vlabs/types.go#L39

Reply:
So we shouldn't set it here.

Reply:
got you! thanks :)